src/Umbraco.Core/BackOffice/BackOfficeIdentityUser.cs

```diff
@@ -39,7 +40,7 @@ public class BackOfficeIdentityUser : IdentityUser<int, IIdentityUserLogin, Iden
         /// <param name="email">This is allowed to be null (but would need to be filled in if trying to persist this instance)</param>
         /// <param name="culture"></param>
         /// <returns></returns>
-        public static BackOfficeIdentityUser CreateNew(IGlobalSettings globalSettings, string username, string email, string culture)
+        public static BackOfficeIdentityUser CreateNew(GlobalSettings globalSettings, string username, string email, string culture)
```

Comment:
I guess this should be  `IOptions<GlobalSettings>`? . Same in other locations?

Reply:
I think this is OK as is.  We only really need `IOptions<GlobalSettings>` if we're injecting them in via the constructor.  For this static factory method, we can just pass in the POCO directly from the calling code (where it's probably been unpacked from `IOptions<GlobalSettings>`).

Reply:
But... if we want to support hot reloading of config then we need to pass in IOptionsSnapshot (if in request based/transient lifetime) or use IOptionsMonitor, if we pass in the real instance, that instance will never change

Reply:
That's true, though I think this parameter is still OK typed as just `GlobalSettings`.  This `CreateNew` method is called (for example) from `UsersController`, that takes an `IOptions<GlobalSettings>` parameter (and so it's there that, if we wanted to, we could instead use `IOptionsSnapshot<GlobalSettings>`.  This can be unpacked though for calling this `CreateNew` method and so we can just pass the POCO (it doesn't need to be passed with it's `IOptions`, or `IOptionsSnapshot`, wrapper).

See my comment above too about why I've used `IOptions`, at least for now, over the other two possibilities (starts: _"I've done this with IOptions over IOptionsSnapshot partly..."_).